### PR TITLE
Fix an unhandled exception when the first chunk doesn't contain (0002,0010)

### PR DESCRIPTION
**Pull request by mch**
Also fixes a bug in logger configuration.

### CODE REVIEW

**Comment by grmble** [comment]
This seems like a high quality contribution, there are even tests.

But I wonder about the motivation.  When exactly does the problem happen?  Because I have trouble to see from the test what the real world conditions are that lead to the problem.

**Comment by grmble** [comment]
Never mind, I see it now.

I was afraid you were trying to decode a File Metainfo with missing Transfer Syntax UID and wanted to suppress the resulting error :)

File: src/decoder.coffee [hunk 1]
```diff
@@ -100,11 +100,12 @@ class Decoder extends stream.Transform
...
       @metainfo_done = false
 
-      @metainfo_listener = (event) =>
-        if event.element.tag == 0x00020010
-          @metainfo_ts = event.vr.value()
-          log.debug {ts: @metainfo_ts}, "metainfo transfer syntax found"
-      @on 'data', @metainfo_listener
+      if not @metainfo_listener
+        @metainfo_listener = (event) =>
+          if event && event.element && event.element.tag == 0x00020010
```
Comment (grmble) [inline src/decoder.coffee]:
```suggestion
          if event.element.tag == 0x00020010
```

I like exceptions when things are not right.

I think the event registration (and de-registration) is correct now, but I would prefer an exception to silent malfunction.

Comment (mch) [reply src/decoder.coffee]: Ok, that's reasonable. This throwing lead me to the underlying problem after all.

**Pull request merged**